### PR TITLE
Update GNOME runtime to version 47

### PR DESCRIPTION
**Pull request by yakushabb**
### Update GNOME runtime to version 47
- GNOME runtime version 45 has reached EOL.

### Fix appdata papercuts

### CODE REVIEW

**Comment by yakushabb** [comment]
@SilverRainZ ready for review.

File: fix-appdata.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+-    <content_attribute id="language-discrimination">none</content_attribute>
+-    <content_attribute id="social-chat">intense</content_attribute>
+-    <content_attribute id="social-info">none</content_attribute>
+-    <content_attribute id="social-audio">none</content_attribute>
+-    <content_attribute id="social-location">none</content_attribute>
+-    <content_attribute id="social-contacts">none</content_attribute>
+-    <content_attribute id="money-purchasing">none</content_attribute>
+-    <content_attribute id="money-gambling">none</content_attribute>
+-  </content_rating>
++  <content_rating type="oars-1.1"/>
```
Comment (SilverRainZ) [inline fix-appdata.patch:47]: I am unfamiliar with XDG's metadata, can you please give any reference about these changes?

**Comment by SilverRainZ** [comment]
B.T.W., do you mind also pushing the metadata changes to Srain's repo?

File: fix-appdata.patch (hunk 1 above)
Comment (yakushabb) [reply fix-appdata.patch:47]: You don't need to mark them as 'none'; the default is already set to 'none'.

**Review by SilverRainZ — APPROVED** [review]
Thank you!

**Pull request merged**